doc/_scripts/software_maturity/software_maturity_features.yaml

```diff
@@ -104,6 +104,14 @@ features:
         - SHIELD_NRF7002EB
         - BOARD_NRF7002DK_NRF5340_CPUAPP
         - BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001
+    Thread Co-existence:
+      rule: WIFI_NRF700X && NRF700X_THREAD_COEX
+      boards_and_shields:
+        - SHIELD_NRF7002EK
+        - SHIELD_NRF7002EK_NRF7001
+        - SHIELD_NRF7002EB
+        - BOARD_NRF7002DK_NRF5340_CPUAPP
```

Comment:
Presently it has been tested only on `BOARD_NRF7002DK_NRF5340_CPUAPP` - please remove others from the supported list. We'll add once we have tested them.